examples/with-uniswap/src/config.ts

```diff
@@ -0,0 +1,39 @@
+import * as dotenv from "dotenv";
+import * as path from "path";
+import type { Token } from "@uniswap/sdk-core";
+import { FeeAmount } from "@uniswap/v3-sdk";
+import { Environment, UNI_TOKEN_GOERLI, WETH_TOKEN_GOERLI } from "./constants";
+
+// Load environment variables from `.env.local`
+dotenv.config({ path: path.resolve(process.cwd(), ".env.local") });
```

Comment:
Generally the pattern is to call `dotenv.config()` only once at the entry point file. We can either have that call happen early in `index/main.ts` (before it imports other modules), or change these configs to be lazy -- either way works